source/deprecations/v2.x.html.md

```diff
@@ -615,3 +615,45 @@ export default Router.map(function() {
   });
 });
 ```
+
+#### Enumerable.contains
+
+##### until: 3.0.0
+##### id: ember-runtime.enumerable-contains
+
+The `Enumerable.contains` and `Array.contains` methods were deprecated in favor of `Enumerable.includes` and `Array.includes` 
+to stay in line with ES standards. See [RFC](https://github.com/emberjs/rfcs/blob/master/text/0136-contains-to-includes.md) for details.
+
+`contains` and `includes` have similar behaviours, excepting regarding `NaN` values management because includes uses
```

Comment:
Perhaps:

> `contains` and `includes` have similar behaviors. A notable exception is how `NaN` values are handled. `contains` uses [strict equality comparison](https://tc39.github.io/ecma262/2016/#sec-strict-equality-comparison) for testing inclusion while `includes` uses the [SameValueZero algorithm](https://tc39.github.io/ecma262/2016/#sec-samevaluezero).

* US-centric "behavior" 😬 
* Simplify the sentences a bit
* First describe the old behavior then the new